api-contracts/openapi/paths/workflow/workflow.yaml

```diff
@@ -366,6 +366,79 @@ workflowRuns:
     summary: Get workflow runs
     tags:
       - Workflow
+workflowRunsMetrics:
+  get:
+    x-resources: ["tenant"]
+    description: Get all workflow runs for a tenant
+    operationId: workflow-run:metrics
```

Comment:
This doesn't really matter at the moment, and isn't documented, but this should be `workflow-run:get:metrics`, the second part of the operation is always a verb in the other endpoints. This will eventually be important when doing RBAC.

Reply:
good catch